File: src/main/java/cc/mirukuneko/realtrainmodrenewed/item/SelectableModelInfo.java
package cc.mirukuneko.realtrainmodrenewed.item;

public record SelectableModelInfo(String id, String displayName, String packName, String buttonTexture, String category) {
    public SelectableModelInfo(String id, String displayName, String packName, String buttonTexture) {
        this(id, displayName, packName, buttonTexture, packName);
    }
}

File: src/main/kotlin/cc/mirukuneko/realtrainmodrenewed/item/SelectableModelInfo.kt
// SPDX-License-Identifier: LGPL-3.0-or-later
// Copyright © 2026 mirukuneko and RealTrainModRenewed contributors
package cc.mirukuneko.realtrainmodrenewed.item

data class SelectableModelInfo @JvmOverloads constructor(
    val id: String,
    val displayName: String,
    val packName: String,
    val buttonTexture: String,
    val category: String = packName,
)
